coconut/control/control.go

```diff
@@ -250,14 +260,49 @@ func GetEnvironments(cxt context.Context, rpc *coconut.RpcClient, cmd *cobra.Com
 }
 
 func CreateEnvironment(cxt context.Context, rpc *coconut.RpcClient, cmd *cobra.Command, args []string, o io.Writer) (err error) {
-	wfPath, err := cmd.Flags().GetString("workflow-template")
+	configPayload, err := cmd.Flags().GetString("configuration")
+	if err != nil {
+		err = errors.New("cannot get configuration payload value")
+		return
+	}
+	if len(configPayload) == 0 {
+		err = errors.New("cannot create environment with empty configuration payload")
+		return
+	}
+	configPayloadDoc, err := os.ReadFile(configPayload)
+	if err != nil {
+		err = errors.New("cannot read configuration payload file")
+		return
+	}
+	payloadData := new(ConfigurationPayload)
+	err = json.Unmarshal(configPayloadDoc, &payloadData)
 	if err != nil {
+		err = errors.New("cannot unmarshal configuration payload")
 		return
 	}
-	if len(wfPath) == 0 {
-		err = errors.New("cannot create empty environment")
+
+	var wfPath string
+	userWfPath, err := cmd.Flags().GetString("workflow-template")
```

Comment:
... and therefore the actual WFT path can come either from the configuration payload, or from `-w`.